integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/ReplaceKeys.feature

```diff
@@ -1,14 +1,13 @@
-@SmartMetering @Platform @SmartMeteringConfiguration @NightlyBuildOnly
+@SmartMetering @Platform @SmartMeteringConfiguration @NightlyBuildOnly @ReplaceKeys
```

Comment:
Are these Key specific annotations intentionally?

Reply:
Yes, to be able to test key replacement specifically